### DISCUSSION TITLE
Problem with DAV integration for calendar in Outlook

### DISCUSSION THREAD

**Discussion by bfeddes** [YetiForceCompany/YetiForceCRM#17072] [Ignored/Incomplete]
I already read a discussion where a different Yeti user had this issue, and that discussion was unable to resolve my problem. I attached some screenshots along with a description of what isn't working.

Yetiforce Key setup:
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/112420814/d83684f7-4214-492a-9166-35a303c8753f)

My configuration using CalDav Synchronizer on Outlook works and says the connection test was successful, as shown below:
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/112420814/e8707fc7-2ace-4695-a0bf-7b63c8acf91a)

However, when I try to integrate I get this error: 
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/112420814/462e94ea-503f-487a-a50c-019c609d89a1)
All errors are the same, and there is 4 of them. "System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. --> System.NullReferenceException: Object reference not set to an instance of an object."
at DDay.iCal.iCalDateTime.get_UTC()
etc... as shown in the screenshot.

When I grab a specific .ics file from the dav.php page, that is able to be added to the Outlook calendar, but the automatic synchronization does not work.

Some steps I have already taken as instructed by the other discussions I read:
1. change from false to "Private" in the /Components/Dav.php folder as shown below:
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/112420814/500ae9bb-6619-47fd-8dc3-6d5a4b5737ab)

Beyond that from what I read, nothing has helped. Was hoping someone with more knowledge could help me out as this is a feature our company really wants to implement.

**Comment by mariuszkrzaczkowski** [comment]
bug/error >> https://github.com/YetiForceCompany/YetiForceCRM/issues/new?assignees=&labels=&template=1-bug-report.md&title=%5Bbug%5D+
question >> https://github.com/YetiForceCompany/YetiForceCRM/discussions

https://github.com/YetiForceCompany/YetiForceCRM/issues/13908

you want to solve the problem, you want to help someone, attach all the required information, especially the **LOGS**
